build/XCBuildData/fc98efb9a669f35cfc6243fbfcb55a1f.xcbuilddata/build-request.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Hello, I don't want to block the PR any longer, but may I know what's the deal with this build folder?
How are these being generated?
I don't think these should be here.

Reply:
Yeah, unfortunately these files should definitely not be here.

Reply:
Thanks for catching that! Yes, the `build/ `folder is generated during the build process and shouldn’t be in the repo. I’ve updated the `.gitignore` to prevent committing it in future.